app/src/components/TeamPayroll/PaidSalariesChart.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
 import { Box } from '@aragon/ui'
+import SalaryChart from '../SalaryChart'
+import { useAppState } from '@aragon/api-react'
 
 function PaidSalariesChart() {
+  const { payments } = useAppState()
```

Comment:
should we call this `useAppState()` inside the `<SalaryChart />` component? since seems that we are not using it here

Reply:
Yeah good point

Reply:
Ah actually we do use it in `MySalaryChart` which renders the same `<SalaryChart />` component.